library/cartridge_validate_config.py

```diff
@@ -11,9 +11,28 @@
     'hostvars': {'required': True, 'type': 'dict'}
 }
 
-INSTANCE_REQUIRED_PARAMS = ['advertise_uri']
-INSTANCE_FORBIDDEN_PARAMS = ['alias', 'console_sock', 'pid_file', 'workdir', 'cluster_cookie']
-
+INSTANCE_REQUIRED_PARAMS = ['cartridge_app_name', 'cartridge_cluster_cookie', 'config']
+PARAMS_THE_SAME_FOR_ALL_HOSTS = [
+    'cartridge_app_name',
+    'cartridge_cluster_cookie',
+    'cartridge_auth',
+    'cartridge_bootstrap_vshard',
+    'cartridge_failover',
+    'cartridge_allow_downgrade',
```

Comment:
Just remember you - https://github.com/tarantool/ansible-cartridge/commit/cd898bd27c5efc16b6faa3aecf8d0116179680fb